client/wfnews-war/src/main/angular/src/app/app.module.ts

```diff
@@ -44,10 +45,16 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { RouteReuseStrategy } from '@angular/router';
 import { ServiceWorkerModule } from '@angular/service-worker';
 import { YouTubePlayerModule } from '@angular/youtube-player';
+import { NotificationSnackbarComponent } from '@app/components/notification-snackbar/notification-snackbar.component';
 import { DialogExitComponent } from '@app/components/report-of-fire/dialog-exit/dialog-exit.component';
 import { RoFReviewPage } from '@app/components/report-of-fire/review-page/rof-review-page.component';
+import { notificationMapComponent } from '@app/components/saved/add-saved-location/notification-map/notification-map.component';
```

Comment:
I know you're not introducing this here but I think this component name should be capitialised so there is uniformity with the rest of the components in the project.